### ISSUE TITLE
Should I expect something from the all session button?

### ISSUE THREAD

**Issue by uhlhosting** [manland/mattermost-plugin-analytics#33]
<img width="625" alt="image" src="https://user-images.githubusercontent.com/6324047/56532262-bbdd9f80-6555-11e9-8ff3-47a6d8547478.png">

What is the role of the all sessions line chart, looks like a button, yet it cannot be pressed.

**Comment by manland** [comment]
Its an aggregate chart of all your stored sessions, with nb messages by channel over the time.

You can see an example on the screenshot : https://github.com/manland/mattermost-plugin-analytics/blob/master/screenshot.png

But I guess you haven't session stored (because its the first week). I need to add a check!

Thanks to report this minor issue

**Comment by tgly307** [comment]
@manland hi, thanks for the great plugin.
I have a question. What do the stored sessions mean?

**Comment by manland** [comment]
@tgly307 hi thanks!

Stored sessions are all previous week recorded in kvstore has [analytic](https://github.com/manland/mattermost-plugin-analytics/blob/master/server/analytic.go#L10) : 

```go
type Analytic struct {
	// Start of recording
	Start time.Time
	// End of recording metrics (time.Zero by default)
	End time.Time
	// Channels store number of messages by channels id
	Channels map[string]int64
	// ChannelsReply store number of reply by channels id
	ChannelsReply map[string]int64
	// Users store number of messages by user id
	Users map[string]int64
	// UsersReply store number of reply by user id
	UsersReply map[string]int64
	// FilesNb store number of files uploaded
	FilesNb int64
	// FilesSize store weigth of files uploaded
	FilesSize int64
}
```

So for each week you have nb messages by channel, nb reply by channel, nb files downloaded... And I use all recorded analytic to build a chart of nb messages by channel over time.

I could draw all kind of chart with all this data, but I think messages by channel is the better. If you want other charts just open issue and I will add them ;)

**Comment by tgly307** [comment]
@manland thanks for reply, I get it.

**Comment by uhlhosting** [comment]
Also on mobile app in iOS the charts dont show up.
Maybe would be great to change them into values or?

**Comment by manland** [comment]
@uhlhosting yes I should do better alt messaging. And maybe try png version, maybe it will work on mobile app... I test it ASAP ;)
